specs/binding-filesystem.spec/src/main/resources/META-INF/zilla/filesystem.idl

```diff
@@ -18,7 +18,10 @@ scope filesystem
     {
         READ_PAYLOAD,
         READ_EXTENSION,
-        READ_CHANGES
+        READ_CHANGES,
+        WRITE_PAYLOAD,
+        WRITE_EXTENSION,
+        WRITE_CHANGES
```

Comment:
Not sure if we need a separate `WRITE_EXTENSION`, is there a use case for that from `http-filesystem` binding for example?